js/llm.ts

```diff
@@ -125,8 +125,6 @@ export async function OpenAIClassifier<RenderArgs, Output>(
     content: m.content && mustache.render(m.content as string, renderArgs),
   }));
 
-  let ret = null;
-  let validityScore = 1;
```

Comment:
I noticed we weren't logging anything in JS, so I just removed it from JS and python. If you'd like me to re-enable it in both languages as an option, I can do that too.